js/data.js

```diff
@@ -6,3 +6,12 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var jsonData = JSON.stringify(data);
+  this.localStorage.setItem('jsonData', jsonData);
+});
+
+if (localStorage.getItem('jsonData') === JSON.stringify(data)) {
```

Comment:
Our condition should just check if there is JSON data stored in local storage. 

This would be accomplished with:

```
if (localStorage.getItem('jsonData')) {
  // code to execute
}  

There is no need for a comparison to take place in the conditional. It will be truthy or falsy.